docs/ops/logical/LogicalNot_1.md

```diff
@@ -22,15 +26,8 @@
 
 * *T*: boolean type.
```

Comment:
Please apply this [comment](https://github.com/openvinotoolkit/openvino/pull/6852#discussion_r683295562) here for consistency